users.js

```diff
@@ -95,6 +96,48 @@ var lockedUsers = Object.create(null);
 var lockedRanges = Users.lockedRanges = Object.create(null);
 var rangelockedUsers = Object.create(null);
 
+function importLocksBans() {
+	fs.readFile('config/locksbans.txt', function (err, data) {
+		if (err) return;
+		data = ('' + data).split("\n");
+		for (var i = 0; i < data.length; i++) {
+			if (!data[i]) continue;
+			var row = data[i].split("|");
+			var entry = {
+				time: parseInt(row[1]),
+				userid: row[2]
+			};
+			if (row[0] === 'lock') {
+				var table = (row[4] ? lockedIps : lockedUsers);
+				table[row[3]] = entry;
+			} else if (row[0] === 'ban') {
+				var table = (row[4] ? bannedIps : bannedUsers);
+				table[row[3]] = entry;
+			}
+		}
+	});
+}
+function exportLocksBans() {
+	var buffer = '';
+	for (var ip in bannedIps) {
+		if (!bannedIps[ip].time) continue;
+		buffer += "ban|" + bannedIps[ip].time + "|" + bannedIps[ip].userid + "|" + ip + "|ip\n";
```

Comment:
Maybe make the first value <code>banip</code> rather than splitting the "ip" part as a separate value?
(Same for locks)